src/static/js/ace2_inner.js

```diff
@@ -3858,7 +3858,7 @@ function Ace2Inner() {
     // Apply height to existing sidediv lines
     currentLine = 0;
     while (sidebarLine && currentLine <= lineNumbersShown) {
-      if (lineHeights[currentLine]) {
+      if (lineHeights[currentLine] !== undefined) {
```

Comment:
This is more idiomatic:

```suggestion
      if (lineHeights[currentLine] != null) {
```

Reply:
By definition, an `Array` returns `undefined` if the given index can not be found. So I think `undefined`  is the more suitable option.

Reply:
Yes it returns `undefined`, but loose comparison with `null` is more idiomatic (with good reasons) and matches the style of the rest of Etherpad (most of it anyway).